### DISCUSSION TITLE
Image Weights

### DISCUSSION THREAD

**Discussion by radical576** [willwulfken/MidJourney-Styles-and-Keywords-Reference#18] [General]
I am fascinated by negative image weights. I was intrigued when I noticed high consistency between both prompts and images when using `--iw -1` . I am not able to rplicate these results and get very different results between the different prompts and test images. I am not sure if the model was changed or something else is going on like I am not entering the prompt correctly
IW page:
https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/blob/main/Pages/MJ_V3/Summary_Pages/Image_Weight_Comparison.md

sphere
`https://s.mj.run/TTqVVSudxZg Mandlebulb --seed 4776 --iw -1`
![image](https://user-images.githubusercontent.com/6817685/192331957-65c3eb1f-3ef2-4ad2-a2a2-7d8a064c1c7c.png)

sunset
`https://s.mj.run/LduV0i9H-5M Mandlebulb --seed 4776 --iw -1`
![image](https://user-images.githubusercontent.com/6817685/192332015-bacfde34-8fce-4caf-8cc5-9abe5c7e761e.png)

**Comment by willwulfken** [comment]
That's interesting. I'm getting the same result as you. I'm not really sure... I may need to do some experimentation
